package.json

```diff
@@ -13,6 +13,9 @@
     "docpad-plugin-stylus": "~2.10.0",
     "docs-core": "fluid-project/docs-core"
   },
+  "peerDependencies": {
+    "docpad": "6.79.4"
+  },
```

Comment:
Rather than using a peer dependency why not add docked as a regular dependency, pinned to the required version. Then add an NPM script like below:

```
  "scripts": {
      "docpad": "node ./node_modules/docpad/bin/docpad run"
  },
```

In this way, the user wouldn't have to have a globally installed docpad and could just run `npm run docpad` to launch it. Although you should make sure it works cross platform. You could add similar scripts for the other docpad commands that are needed too.